### PR TITLE
Project Logos fixed for TABLET Size Screens

### LINKED ISSUE #881: Responsive Problem with Project Logos
**Describe the bug**
When reducing the size of the web browser or the mobile view. The logos of the projects over-flows of their respective container (box). 

**To Reproduce**
1. Reduce the size of the browser or check-in mobile view

**Expected behavior**
Logos should stay within their container.

**Screenshots**
image

**Desktop (please complete the following information):**
 - OS: macOS Catalina
 - Browser: Google Chrome
 - Version: 10.15.3

**Smartphone (please complete the following information):**
NA

**Additional context**
NA

### PR DESCRIPTION
**Pull request by suyashchauhan**
## Purpose

The purpose of this PR is to fix #881 

## Approach
1. Changed the width to now be the max-width so that the image doesn't overflow out of its container . 
2. added media query for tablet which removes the left and right padding 
### Screenshots
Before : 
before

After : 
after

  
### Preview Link

https://pr-883-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Test environment
 Chrome (Version 88.0.4324.190) , Firefox(Version 86.0) on  Windows(Version 1909)

### CODE REVIEW

**Review by anjula-sack — CHANGES_REQUESTED** [review]
Can we divide the card into two rows and show them like this?
Because even the logos are fixed those cards are look cluttered.
image

**Comment by suyashchauhan** [comment]
> Can we divide the card into two rows and show them like this?

yes that looks good so do you want this for all the screen sizes or just for tablets (between 576 and 768 px)

**Comment by anjula-sack** [comment]
> > Can we divide the card into two rows and show them like this?
> 
> yes that looks good so do you want this for all the screen sizes or just for tablets (between 576 and 768 px)

only for tablets

**Comment by suyashchauhan** [comment]
> only for tablets

Done

File: assets/css/global-custom.css [hunk 1]
```diff
@@ -594,3 +602,12 @@ body{
...
 }
+@media (min-width:576px) and (max-width:768px) {
+    #projects>div>div:nth-child(2)>a:nth-child(3) {
+        display: none;
+    }
+
+    #projects>div>div:nth-child(4)>a:nth-child(1) {
+        display: block;
+    }
+}
```
Comment (anjula-sack) [inline assets/css/global-custom.css]: Please add a line here

File: index.html [hunk 2]
```diff
@@ -167,6 +167,21 @@ <h5 class="display-3 text-center project-stat">100+</h5>
             </div>
             <br/>
             <div class="card-deck justify-content-center col-md-12">
+                <a href="irc.html" class="card col-md-4 shadow project-card">
```
Comment (anjula-sack) [inline index.html]: In every card use `col-lg-4` instead of `col-md-4`

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: assets/css/global-custom.css (hunk 1 above)
Comment (suyashchauhan) [reply assets/css/global-custom.css]: ✅

File: index.html (hunk 2 above)
Comment (suyashchauhan) [reply index.html]: ✅

**Review by anjula-sack — APPROVED** [review]
Thank you! @suyashchauhan

**Pull request merged**